src/openseadragon.js

```diff
@@ -198,6 +198,10 @@
   *     'destination-over', 'destination-atop', 'destination-in',
   *     'destination-out', 'lighter', 'copy' or 'xor'
   *
+  * @property {Boolean} [imageSmoothingEnabled=true]
+  *     Image smoothing for canvas rendering (only if canvas is used). Not every browser
```

Comment:
"Not every browser is compatible" sounds like it might be dangerous. Maybe change to something like "Note: Ignored on some browsers".